builtin/providers/azurerm/resource_arm_subnet_test.go

```diff
@@ -171,3 +257,108 @@ resource "azurerm_route" "test" {
 	next_hop_in_ip_address = "10.10.1.1" 
 }
 `
+
+var testAccAzureRMSubnet_routeTable = `
```

Comment:
FYI for newer test config's we're tending to make these functions which take an Int and handle formatting the string, rather than spreading the string format throughout the test functions (given we're now using them in multiple places), i.e.
```
func testAccAzureRMSubnet_routeTable(rInt int) string {
 return fmt.Sprintf(....., rInt,rInt)
}
```